egs/heroico/s5/local/subs_download.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2017 John Morgan
+# Apache 2.0.
+
+tmpdir=data/local/tmp
+download_dir=$(pwd)
+mkdir -p $download_dir
+subs_src="http://opus.lingfil.uu.se/download.php?f=OpenSubtitles2018/en-es.txt.zip"
```

Comment:
Again, please make this an arg passeed in from the run.sh so it's obvious to the user where the data comes from.